safesql.go

```diff
@@ -38,21 +50,51 @@ func main() {
 	c := loader.Config{
 		FindPackage: FindPackage,
 	}
-	c.Import("database/sql")
 	for _, pkg := range pkgs {
 		c.Import(pkg)
 	}
 	p, err := c.Load()
+
+	imports := GetImports(p)
+
+	if _, exist := imports["database/sql"]; exist {
+		if verbose {
+			fmt.Println("Enabling support for database/sql")
+		}
+		config.DatabaseSql = true
+	}
+
+	if _, exist := imports["github.com/jmoiron/sqlx"]; exist {
+		if verbose {
+			fmt.Println("Enabling support for sqlx")
+		}
+		config.Sqlx = true
+	}
+
+	if !(config.Sqlx || config.DatabaseSql) {
+		fmt.Printf("No packages in %v include a supported database driver", pkgs)
```

Comment:
To be honest, I'd probably just skip the whole import checking business. If a package doesn't use `database/sql`, it doesn't have `database/sql` injection vulnerabilities, end of story (likewise for `sqlx`). This utility shouldn't claim to protect all uses of all libraries; nor should it error out if someone decides to run `safesql` over all of their binaries, even those that don't require a database.